src/main/kotlin/coverosR3z/timerecording/TimeRecordingUtilities.kt

```diff
@@ -99,7 +99,8 @@ class TimeRecordingUtilities(private val persistence: ITimeEntryPersistence, pri
         return persistence.readTimeEntriesOnDate(employee, date)
     }
 
-    override fun getAllEntriesForEmployee(employee: Employee): List<TimeEntry> {
+    override fun getAllEntriesForEmployee(employeeId: Int): List<TimeEntry> {
+        val employee = persistence.getEmployeeById(employeeId)
```

Comment:
I wonder if this should still have the abiity to take the employee as an argument, too